Website/src/rendering/src/components/Forms/AddressBookForm.tsx

```diff
@@ -23,14 +23,31 @@ const AddressBookForm = (props: AddressBookFormProps): JSX.Element => {
   const [address, setAddress] = useState(props?.address);
   const [isAddressValid, setIsAddressValid] = useState(false);
   const [defaultBilling, setDefaultBilling] = useState(
```

Comment:
Just a minor thing, sometimes when editing a default shipping or billing (or both) address via the address book, the checkboxes do not appear as checked.